src/runtime/contrib/json/json_runtime.h

```diff
@@ -183,7 +183,10 @@ class JSONRuntimeBase : public ModuleNode {
       uint32_t nid = input_nodes_[i];
       std::string name = nodes_[nid].name_;
       if (nodes_[nid].op_type_ == "input") {
-        input_var_idx_.push_back(nid);
+        CHECK_EQ(nodes_[nid].GetOpShape().size(), nodes_[nid].GetOpDataType().size());
+        for (size_t j = 0; j < nodes_[nid].GetOpShape().size(); ++j) {
+          input_var_idx_.push_back(EntryID(nid, j));
```

Comment:
By doing so, `input_var_idx_` is not keeping index but entry ID. I'd suggest renaming it to be clearer.